site/docs/guides/migrating-from-sub-builder-interfaces.md

```
@@ -0,0 +1,211 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+     License, v. 2.0. If a copy of the MPL was not distributed with this
+     file, You can obtain one at https://mozilla.org/MPL/2.0/. -->
+---
+title: "Migrating from Sub-Builder Interfaces"
+layout: simple
+---
+
+# Migrating from Sub-Builder Interfaces
```

markdownlint-cli2 (0.23.2)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)